tests/REGRESSIONTEST.md

```diff
@@ -2,24 +2,50 @@
 
-The usage instructions are outputted whenever the script is run. Currently, there are include, exclude, and normtype options. Include is used if a user only wants to run some tests. Exclude is used if a user wants to run all tests except for some. The default is that all tests are run. Without a normtype option, the tests default to l2, but a user can use any normtype or optionally add multiple to run the tests with both l2 and max sequentially, for example.
+The usage instructions are outputted whenever the script is run. Currently, there are include, exclude, and stopping options. Include is used if a user only wants to run some tests. Exclude is used if a user wants to run all tests except for some. The default is that all tests are run. The stop option is used to end the regression tests at the first test failure. This allows a user to look at the simulation data from both the user branch and the baseline branch, whereas without, it would be overwritten by the next test.
 
-Here are some example runs and results:
+How to run the tests on both MAC and LC
 
-./runRegressionTests.sh -> Run all tests with l2 norm.
+1. If you have just changed branches, run "make clean".
 
-./runRegressionTests.sh -i "sedov_blast gresho_vortices" -> Run sedov_blast and gresho_vortices with l2 norm.
+2. Run "make" to make sure you your branch is up-to-date with any local changes.
 
-./runRegressionTests.sh -e "taylor-green" -> Run all tests with l2 norm except taylor-green.
+3. ./tests/runRegressionTests.sh (if in the base directory) or ./runRegressionTests.sh (if in the tests directory). Look below for
+test options.
 
-./runRegressionTests.sh -i "sedov_blast" -e "taylor-green" -> Error. -i and -e can not be used simultaneously.
+4. Test commands/logs are stored in tests/results. Since each run overwrites the previous run, only the last run's data is saved
+in run and tests/Laghos/run. Use option -f to stop at the first failure and look at the failed run's data.
+
+5. To erase the regression test data, run from the base directory: make clean-regtest
+
+How to add a non-time-windowing test
+
+1. Choose an appropriate sub-test directory.
+2. Copy sedov-blast/sedov-blast.sh to your test directory, renaming it as desired.
+3. Your tests will be run both serially and in parallel.
+4. Choose a number for NUM_PARALLEL_PROCESSORS.
+5. The tests will be run sequentially (1, 2, 3, 4, ...). Create as many tests as
+desired following the given format.
+6. All tests should have the format $HEADER laghos ...
+7. Name your tests in testNames, which is an array of test names.
 
-./runRegressionTests.sh -i "triple-point" -n "l1 max" -> Run triple-point with l1 norm, then max norm.
+How to add a time-windowing test
```

Comment:
@kevinhkhuynh Would you cover how to add a parametric test, also? Right now, @siuwuncheung  is working on time-windowing parametric implementation, but the parametric with single time window is already implemented by @dylan-copeland. The example runs can be found in the following confluence page:

https://myconfluence.llnl.gov/display/ROM/Parameter+variation

Reply:
@chldkdtn Which branch should be used to run these tests?

Reply:
@kevinhkhuynh, this branch, i.e., RHS optimization.